demos/smart_classroom_demo/cpp/src/detector.cpp

```diff
@@ -57,7 +57,7 @@ void FaceDetection::submitRequest() {
 
 void FaceDetection::enqueue(const cv::Mat &frame) {
     if (!request) {
-        request = net_.CreateInferRequestPtr();
+        request = std::make_shared<InferenceEngine::InferRequest>(net_.CreateInferRequest());
```

Comment:
Is it possible to replace a ptr to `InferRequest` with actual `InferRequest` in such cases?